packages/dapi/test/integration/grpcServer/handlers/platform/waitForStateTransitionResultHandlerFactory.js

```diff
@@ -241,7 +241,7 @@ describe('waitForStateTransitionResultHandlerFactory', () => {
     expect(resultStoreTreeProof).to.deep.equal(storeTreeProofs);
 
     expect(driveClientMock.fetchProofs).to.be.calledOnceWithExactly({
-      identityIds: stateTransitionFixture.getModifiedDataIds(),
+      identityIds: stateTransitionFixture.getModifiedDataIds().map(Buffer.from),
```

Comment:
It's array of `Identifier` so you just need to call toBuffer() method.

```suggestion
      identityIds: stateTransitionFixture.getModifiedDataIds().map((identifier) => identifier.toBuffer()),
```